aws/logs_monitoring/release.sh

```diff
@@ -26,60 +22,101 @@ if [ -z "$2" ]; then
 elif [[ ! $2 =~ [0-9]+\.[0-9]+\.[0-9]+ ]]; then
     echo "Must use a semantic version, e.g., 3.1.4"
     exit 1
-elif [[ ! "$2" > "$CURRENT_VERSION" ]]; then
-    echo "Must use a version greater than the current ($CURRENT_VERSION)"
-    exit 1
 else
     VERSION=$2
 fi
 
-# Make the template private (default is public) - useful for developers
-if [[ $# -eq 3 ]] && [[ $3 = "--private" ]]; then
-    PRIVATE_TEMPLATE=true
+# Do a production release (default is staging) - useful for developers
+if [[ $# -eq 3 ]] && [[ $3 = "--prod" ]]; then
+    PROD_RELEASE=true
 else
-    PRIVATE_TEMPLATE=false
+    PROD_RELEASE=false
+fi
+
+if [ -z "$AWS_ACCESS_KEY_ID" ]; then
+    echo 'AWS_ACCESS_KEY_ID not set. Are you using aws-vault?'
+    exit 1
+fi
+
+if [ -z "$AWS_SECRET_ACCESS_KEY" ]; then
+    echo 'AWS_SECRET_ACCESS_KEY not set. Are you using aws-vault?'
+    exit 1
+fi
```

Comment:
Just an idea, can we replace all these lines with `aws sts get-caller-identity`, which should fail and give user an error message if no AWS credentials provided? 

Nits, shall we make the error message more generic about missing aws credentials rather than `aws-vault`?